src/AppInstallerCLICore/Commands/SettingsCommand.cpp

```diff
@@ -36,7 +36,35 @@ namespace AppInstaller::CLI
             context.Reporter.Warn() << Resource::String::SettingLoadFailure << std::endl;
             for (const auto& warning : User().GetWarnings())
             {
-                context.Reporter.Warn() << warning << std::endl;
+                auto warn = context.Reporter.Warn();
```

Comment:
>auto [](start = 16, length = 4)

const auto&

Reply:
This can't be auto& because Warn() does not return a reference.